develop/reference/articles/xx-theme-file-reference/00-theme-reference-intro.markdown

```diff
@@ -0,0 +1,241 @@
+# Theme Reference Guide [](id=theme-reference-guide)
+
+A theme is made up of several files. Although most of the files are named after 
+their matching component, you may be unsure of the function of some of these 
+files.
+
+This document explains each file's usage in more detail, so that you will know
+which files to modify and which files to leave untouched.
+
+## Theme Anatomy [](id=theme-anatomy)
+
+There are two main approaches to theme development for @product@ 7: themes built 
+using the Node build tools with the [themes generator](/develop/tutorials/-/knowledge_base/7-0/themes-generator) 
+and themes built using the Plugins SDK.
+
+For themes developed with the themes generator, your theme anatomy is organized
+with the structure shown below. Although themes developed with the Plugins SDK 
+have a slightly different anatomy, the core theme files are the same. Note that 
+the `build` directory is shown for reference, and is not created by default.
+
+- `theme-name/`
+    - `build/`(not created automatically)
+        - `css/`
+            - [`_application.scss`](#application-scss)
+            - [`_aui_custom.scss`](#auicustom-scss)
+            - [`_aui_variables.scss`](#auivariables-scss)
+            - [`_base.scss`](#base-scss)
+            - [`_custom.scss`](#custom-scss)
+            - [`_extras.scss`](#extras-scss)
+            - [`_imports.scss`](#imports-scss)
+            - [`_layout.scss`](#layout-scss)
+            - [`_liferay_custom.scss`](#liferaycustom-scss)
+            - [`_liferay_variables_custom.scss`](#liferayvariablescustom-scss)
+            - [`_liferay_variables.scss`](#liferayvariables-scss)
+            - [`_navigation.scss`](#navigation-scss)
+            - [`_portal.scss`](#portal-scss)
+            - [`_portlet.scss`](#portlet-scss)
+            - [`_taglib.scss`](#taglib-scss)
+            - `application/`
+            - `aui/`
+            - [`aui.scss`](#aui-scss)
+            - `base/`
+            - `layout/`
+            - [`main.scss`](#main-scss)
+            - `navigation/`
+            - `portal/`
+            - `portlet/`
+            - `taglib/`
+        - `templates/`
+            - [`init_custom.ftl`](#initcustom-ftl)
+            - [`init.ftl`](#init-ftl)
+            - [`navigation.ftl`](#navigation-ftl)
+            - [`portal_normal.ftl`](#portalnormal-ftl)
+            - [`portal_pop_up.ftl`](#portalpopup-ftl)
+            - [`portlet.ftl`](#portlet-ftl)
+            - (other directories that have been copied from src)
+    - `dist/` (not created automatically)
+    - [`gulpfile.js`](#gulpfile-js)
+    - [`liferay-theme.json`](#liferay-theme-json)
+    - `node_modules/`
```

Comment:
This directory is generated whenever an `npm install` command is run from the root of the theme, and can be deleted at anytime and re-generated by running `npm install`.